ci/dev/fmt.sh

```diff
@@ -30,7 +30,7 @@ main() {
   doctoc --title '# Contributing' doc/CONTRIBUTING.md > /dev/null
   doctoc --title '# iPad' doc/ipad.md > /dev/null
 
-  if [[ ${CI-} && $(git ls-files --other --modified --exclude-standard) ]]; then
+  if [[ ${CI-} && $(git ls-files --other --modified --exclude-standard | grep -v "lib/vscode") ]]; then
```

Comment:
To satisfy the linter we could do:
```
  if [[ ${CI-} ]] && git ls-files --other --modified --exclude-standard | grep -q -v "lib/vscode"; then
```

Although I think in this case we could actually leave off the `grep -v "lib/vscode"` since this is just checking if any files were modified by `fmt` and nothing in `lib/vscode` will be modified since we excluded those paths above.

Reply:
Ah gotcha! As you can probably tell, I haven't written a lot of shell scripts 😅 

I'll leave it off here!